### PR TITLE
chore(front-end-web): Front End Init

### PR DESCRIPTION
**Pull request by andrewdimmer**

### CODE REVIEW

**Review by jcelynott — APPROVED** [review]

File: front-end-web/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```
Comment (camrongrant1) [inline front-end-web/.gitignore:3]: this is my comment ~ Cam

File: front-end-web/.gitignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+/node_modules
+/.pnp
```
Comment (Darius-Student) [inline front-end-web/.gitignore:5]: First comment!

File: front-end-web/.gitignore (hunk 1 above)
Comment (andrewdimmer) [reply front-end-web/.gitignore:3]: Awesome!

**Review by Darius-Student — APPROVED** [review]

File: front-end-web/.gitignore (hunk 2 above)
Comment (andrewdimmer) [reply front-end-web/.gitignore:5]: Perfect!

Comment (camrongrant1) [reply front-end-web/.gitignore:5]: I was actually the first comment

**Review by BriMcGraw — COMMENTED** [review]
Comment

**Pull request merged**